res/scripts/build.ps1

```diff
@@ -39,7 +39,9 @@ Param(
     [switch]$WhatIf,
     [switch]$Mono,
     [switch]$SkipToolPackageRestore,
-    [switch]$Verbose
+    [switch]$Verbose,
+    [Parameter(ValueFromRemainingArguments = $true)]
```

Comment:
Verbose and ValueFromRemainingArguments do not like each other too much.

Error on command line when running script: <position> : A parameter with the name 'Verbose' was defined multiple times for the command.

Maybe this is because Verbose is a common parameter? A quick test on Win10 shows that VerbosityPreference is set automatically by powershell without having the explicit Verbose functionality in the script.